pytest-embedded-idf/pytest_embedded_idf/dut.py

```diff
@@ -91,6 +91,14 @@ def panic_output_decode_script(self) -> t.Optional[str]:
 
         return os.path.realpath(script_filepath)
 
+    def _get_prefix_map_path(self) -> str:
+        primary = f'{self.app.app_path}/build/gdbinit/prefix_map'
+        fallback = f'{self.app.app_path}/build/prefix_map_gdbinit'
```

Comment:
```suggestion
        primary = os.path.join(self.app.binary_path, 'gdbinit', 'prefix_map')
        fallback = os.path.join(self.app.binary_path, 'prefix_map_gdbinit')
```

This could help resolve the issue when the build directory is not named `build`, and also address OS compatibility issues on Windows.